pkg/capi/infrastructure/infrastructure.go

```diff
@@ -24,7 +24,8 @@ type Provider interface {
 	WatchingNamespace() string
 	Configure(interface{}) error
 	PreInstall() error
-	IsInstalled(ctx context.Context, clientset *kubernetes.Clientset) (bool, error)
+	// TODO: readd ctx as input here once we bump up controller-runtime version.
+	IsInstalled(clientset *kubernetes.Clientset) (bool, error)
```

Comment:
I would suggest to keep using the context in all the places and just skip passing it to underlying kubernetes call for now.

Reply:
It barfed during linting unfortunately :/

Reply:
weird. Would be nice to keep it at least in the interface, but I guess it's fine either way.

Reply:
I can probably tell the linter to ignore. I'll poke at it again once I'm back home.